Lanchonete/Lanchonete/ Features/Menu/Data/MenuManager.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MenuManager.swift
+//  Lanchonete
+//
+//  Created by vera.dias on 2/4/21.
+//
+
+import Foundation
+
+public class MenuManager {
+    func getSandwichTypes () -> [SandwichType] {
+        return SandwichType.allCases
```

Comment:
This is returning something that seems to be accessible without this class, just accessing the `SandwichType`. Is this function necessary? Why not accessing it directly?